.env

```diff
@@ -14,7 +14,7 @@ COGMENT_VERSE_TF_AGENTS_PROMETHEUS_PORT=8001
 COGMENT_VERSE_ENVIRONMENT_PROMETHEUS_PORT=8002
 
 ## Other
-COGMENT_VERSE_MLFLOW_PORT=5000
+COGMENT_VERSE_MLFLOW_PORT=3000
```

Comment:
Please don't change that port 3000 is used by the system in later versions of macOS